src/main/java/org/myrobotlab/service/VirtualArduino.java

```diff
@@ -244,7 +244,12 @@ public void releaseService() {
     if (runner != null) {
       runner.stop();
     }
+    if (uart != null) {
+      uart.releaseService();
+    }
     releasePeers();
+    sleep(300);

```

Comment:
why is there a sleep here?

Reply:
I wish I remembered :P
I'll take it out.